packages/cli/src/Queue.ts

```diff
@@ -30,29 +29,11 @@ export class Queue {
 
 	async init() {
 		const prefix = config.getEnv('queue.bull.prefix');
-		const redisHost = config.getEnv('queue.bull.redis.host');
-		const redisUsername = config.getEnv('queue.bull.redis.username');
-		const redisPassword = config.getEnv('queue.bull.redis.password');
-		const redisPort = config.getEnv('queue.bull.redis.port');
-		const redisDB = config.getEnv('queue.bull.redis.db');
-		// retro-compatibility with redis < 6
-		// prepare new redis options setting in order to define only set values
-		const redisOptions: RedisOptions = {};
-		if (redisHost) {
-			redisOptions.host = redisHost;
-		}
-		if (redisUsername) {
-			redisOptions.username = redisUsername;
-		}
-		if (redisPassword) {
-			redisOptions.password = redisPassword;
-		}
-		if (redisPort) {
-			redisOptions.port = redisPort;
-		}
-		if (redisDB) {
-			redisOptions.db = redisDB;
-		}
+		const redisOptions: RedisOptions = config.getEnv('queue.bull.redis');
```

Comment:
Hi @netroy 
With this commit, you remove my modifications: when the variable `QUEUE_BULL_REDIS_USERNAME` is not set, it will send `username: ''`
Is this going to be a problem ?

Reply:
It doesn't seem to be an issue when I tested it on redis 5 and 6.